avm/res/network/front-door/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -99,10 +99,10 @@ module testDeployment '../../../main.bicep' = [
         {
           name: 'heathProbe'
           properties: {
-            enabledState: ''
-            healthProbeMethod: ''
+            enabledState: 'Enabled'
+            healthProbeMethod: 'HEAD'
```

Comment:
@rozkurt it looks like WAF compliance for PSRule is failing on both waf-aligned and defaults tests.

This update only fixes the waf-aligned test.
To fix the defaults one please add the failing rules to the suppression group for defaults tests, with a comment explaining the reason for the exception. That would be very similar to what already done by the web site module. Ref https://github.com/Azure/bicep-registry-modules/blob/main/avm/utilities/pipelines/staticValidation/psrule/.ps-rule/min-suppress.Rule.yaml#L32